tutorbackup/templates/backup/build/backup/backup_local_services.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+mysqldump --all-databases --single-transaction --quick --quote-names --max-allowed-packet=16M \
+--password=$MYSQL_ROOT_PASSWORD --user=$MYSQL_ROOT_USERNAME --host=$MYSQL_HOST --port=$MYSQL_PORT > /data/mysql_dump.sql
```

Comment:
Micro-nit: Please indent the second line with two spaces here.

Reply:
Will do